### PR TITLE
[1/n] Add verifier for EXIR Aten dialect 

### PR DESCRIPTION
**Pull request by qihqi**
Fixes #ISSUE_NUMBER

### CODE REVIEW

File: test/fx/test_verifier.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+        y = self.relu(y)
+        y = self.linear2(y)
+        y = self.dropout2(y)
+        return y
+
+
+
+class VerifierTest(TestCase):
+
+    def test_exir_verifier(self) -> None:
```
Comment (zhxchen17) [inline test/fx/test_verifier.py:117]: do we need to mention `exir`? I feel like it's just an internal name, but when published we should have a different name.

**Comment by zhxchen17** [comment]
I'm okay with the change in general.

File: test/fx/test_verifier.py (hunk 1 above)
Comment (qihqi) [reply test/fx/test_verifier.py]: done. removed all mentions to exir.

**Review by zhxchen17 — APPROVED** [review]

**Comment by zhxchen17** [comment]
cc @SherlockNoMad

**Pull request closed without merging**